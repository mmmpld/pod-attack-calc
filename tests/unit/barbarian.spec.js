import breakpoints from './breakpoints.js'
const characterId = 2;

describe('Barbarian default values', () => {
    it('standard attack', async () => {
        await breakpoints('11', '2.27', { charactersSelected: characterId });
    })

    it('cleave', async () => {
        await breakpoints('10', '2.5', { charactersSelected: characterId, skillsSelected: 33, weaponsPrimarySelected: 25, iasWeaponPrimary: 105 });
    })

    it('cleave wolf', async () => {
        await breakpoints('5', '5', { charactersSelected: characterId, shapeShiftFormsSelected: 2, skillsSelected: 33, weaponsPrimarySelected: 25, iasWeaponPrimary: 105, werewolfSkillIas: 66 });
    })

    it('frenzy (first swing misses) 2x phaseblade', async () => {
        await breakpoints('8.5', '2.94', { charactersSelected: characterId, skillsSelected: 17, weaponsPrimarySelected: 191, weaponsSecondarySelected: 191 });
    })
})