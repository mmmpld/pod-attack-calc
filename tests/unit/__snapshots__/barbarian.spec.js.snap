// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Barbarian default values cleave 1`] = `
<tbody>
  <tr class="">
    <td>0</td>
    <td>15</td>
    <td>1.66</td>
  </tr>
  <tr class="">
    <td>9</td>
    <td>14</td>
    <td>1.78</td>
  </tr>
  <tr class="">
    <td>18</td>
    <td>13</td>
    <td>1.92</td>
  </tr>
  <tr class="">
    <td>30</td>
    <td>12</td>
    <td>2.08</td>
  </tr>
  <tr class="">
    <td>48</td>
    <td>11</td>
    <td>2.27</td>
  </tr>
  <tr class="highlight-current">
    <td>75</td>
    <td>10</td>
    <td>2.5</td>
  </tr>
  <tr class="">
    <td>125</td>
    <td>9</td>
    <td>2.77</td>
  </tr>
</tbody>
`;

exports[`Barbarian default values cleave wolf 1`] = `
<tbody>
  <th rowspan="52"><span class="vertical-label">Primary Weapon IAS</span></th>
  <tr class="">
    <th>0</th>
    <td>13</td>
    <td>13</td>
    <td>13</td>
    <td>13</td>
    <td>13</td>
    <td>13</td>
    <td>13</td>
    <td>13</td>
    <td>13</td>
    <td>13</td>
    <td>13</td>
    <td>13</td>
    <td>13</td>
    <td>13</td>
    <td>13</td>
    <!---->
  </tr>
  <tr class="">
    <th>5</th>
    <td>12</td>
    <td>12</td>
    <td>12</td>
    <td>12</td>
    <td>12</td>
    <td>12</td>
    <td>12</td>
    <td>12</td>
    <td>12</td>
    <td>12</td>
    <td>12</td>
    <td>12</td>
    <td>12</td>
    <td>12</td>
    <td>12</td>
    <!---->
  </tr>
  <tr class="">
    <th>10</th>
    <td>11</td>
    <td>11</td>
    <td>11</td>
    <td>11</td>
    <td>11</td>
    <td>11</td>
    <td>11</td>
    <td>11</td>
    <td>11</td>
    <td>11</td>
    <td>11</td>
    <td>11</td>
    <td>11</td>
    <td>11</td>
    <td>11</td>
    <!---->
  </tr>
  <tr class="">
    <th>15</th>
    <td>10</td>
    <td>10</td>
    <td>10</td>
    <td>10</td>
    <td>10</td>
    <td>10</td>
    <td>10</td>
    <td>10</td>
    <td>10</td>
    <td>10</td>
    <td>10</td>
    <td>10</td>
    <td>10</td>
    <td>10</td>
    <td>10</td>
    <!---->
  </tr>
  <tr class="">
    <th>20</th>
    <td>10</td>
    <td>10</td>
    <td>10</td>
    <td>10</td>
    <td>10</td>
    <td>10</td>
    <td>10</td>
    <td>10</td>
    <td>10</td>
    <td>10</td>
    <td>10</td>
    <td>10</td>
    <td>10</td>
    <td>10</td>
    <td>10</td>
    <!---->
  </tr>
  <tr class="">
    <th>25</th>
    <td>9</td>
    <td>9</td>
    <td>9</td>
    <td>9</td>
    <td>9</td>
    <td>9</td>
    <td>9</td>
    <td>9</td>
    <td>9</td>
    <td>9</td>
    <td>9</td>
    <td>9</td>
    <td>9</td>
    <td>9</td>
    <td>9</td>
    <!---->
  </tr>
  <tr class="">
    <th>30</th>
    <td>9</td>
    <td>9</td>
    <td>9</td>
    <td>9</td>
    <td>9</td>
    <td>9</td>
    <td>9</td>
    <td>9</td>
    <td>9</td>
    <td>9</td>
    <td>9</td>
    <td>9</td>
    <td>9</td>
    <td>9</td>
    <td>9</td>
    <!---->
  </tr>
  <tr class="">
    <th>35</th>
    <td>9</td>
    <td>9</td>
    <td>9</td>
    <td>9</td>
    <td>9</td>
    <td>9</td>
    <td>9</td>
    <td>9</td>
    <td>9</td>
    <td>9</td>
    <td>9</td>
    <td>9</td>
    <td>9</td>
    <td>9</td>
    <td>9</td>
    <!---->
  </tr>
  <tr class="">
    <th>40</th>
    <td>9</td>
    <td>9</td>
    <td>9</td>
    <td>9</td>
    <td>9</td>
    <td>9</td>
    <td>9</td>
    <td>9</td>
    <td>9</td>
    <td>9</td>
    <td>9</td>
    <td>9</td>
    <td>9</td>
    <td>9</td>
    <td>9</td>
    <!---->
  </tr>
  <tr class="">
    <th>45</th>
    <td>9</td>
    <td>9</td>
    <td>9</td>
    <td>9</td>
    <td>9</td>
    <td>9</td>
    <td>9</td>
    <td>9</td>
    <td>9</td>
    <td>9</td>
    <td>9</td>
    <td>9</td>
    <td>9</td>
    <td>9</td>
    <td>9</td>
    <!---->
  </tr>
  <tr class="">
    <th>50</th>
    <td>8</td>
    <td>8</td>
    <td>8</td>
    <td>8</td>
    <td>8</td>
    <td>8</td>
    <td>8</td>
    <td>8</td>
    <td>8</td>
    <td>8</td>
    <td>8</td>
    <td>8</td>
    <td>8</td>
    <td>8</td>
    <td>8</td>
    <!---->
  </tr>
  <tr class="">
    <th>55</th>
    <td>8</td>
    <td>8</td>
    <td>8</td>
    <td>8</td>
    <td>8</td>
    <td>8</td>
    <td>8</td>
    <td>8</td>
    <td>8</td>
    <td>8</td>
    <td>8</td>
    <td>8</td>
    <td>8</td>
    <td>8</td>
    <td>8</td>
    <!---->
  </tr>
  <tr class="">
    <th>60</th>
    <td>8</td>
    <td>8</td>
    <td>8</td>
    <td>8</td>
    <td>8</td>
    <td>8</td>
    <td>8</td>
    <td>8</td>
    <td>8</td>
    <td>8</td>
    <td>8</td>
    <td>8</td>
    <td>8</td>
    <td>8</td>
    <td>8</td>
    <!---->
  </tr>
  <tr class="">
    <th>65</th>
    <td>7</td>
    <td>7</td>
    <td>7</td>
    <td>7</td>
    <td>7</td>
    <td>7</td>
    <td>7</td>
    <td>7</td>
    <td>7</td>
    <td>7</td>
    <td>7</td>
    <td>7</td>
    <td>7</td>
    <td>7</td>
    <td>7</td>
    <!---->
  </tr>
  <tr class="">
    <th>70</th>
    <td>7</td>
    <td>7</td>
    <td>7</td>
    <td>7</td>
    <td>7</td>
    <td>7</td>
    <td>7</td>
    <td>7</td>
    <td>7</td>
    <td>7</td>
    <td>7</td>
    <td>7</td>
    <td>7</td>
    <td>7</td>
    <td>7</td>
    <!---->
  </tr>
  <tr class="">
    <th>75</th>
    <td>7</td>
    <td>7</td>
    <td>7</td>
    <td>7</td>
    <td>7</td>
    <td>7</td>
    <td>7</td>
    <td>7</td>
    <td>7</td>
    <td>7</td>
    <td>7</td>
    <td>7</td>
    <td>7</td>
    <td>7</td>
    <td>7</td>
    <!---->
  </tr>
  <tr class="">
    <th>80</th>
    <td>6</td>
    <td>6</td>
    <td>6</td>
    <td>6</td>
    <td>6</td>
    <td>6</td>
    <td>6</td>
    <td>6</td>
    <td>6</td>
    <td>6</td>
    <td>6</td>
    <td>6</td>
    <td>6</td>
    <td>6</td>
    <td>6</td>
    <!---->
  </tr>
  <tr class="">
    <th>85</th>
    <td>6</td>
    <td>6</td>
    <td>6</td>
    <td>6</td>
    <td>6</td>
    <td>6</td>
    <td>6</td>
    <td>6</td>
    <td>6</td>
    <td>6</td>
    <td>6</td>
    <td>6</td>
    <td>6</td>
    <td>6</td>
    <td>6</td>
    <!---->
  </tr>
  <tr class="">
    <th>90</th>
    <td>6</td>
    <td>6</td>
    <td>6</td>
    <td>6</td>
    <td>6</td>
    <td>6</td>
    <td>6</td>
    <td>6</td>
    <td>6</td>
    <td>6</td>
    <td>6</td>
    <td>6</td>
    <td>6</td>
    <td>6</td>
    <td>6</td>
    <!---->
  </tr>
  <tr class="">
    <th>95</th>
    <td>6</td>
    <td>6</td>
    <td>6</td>
    <td>6</td>
    <td>6</td>
    <td>6</td>
    <td>6</td>
    <td>6</td>
    <td>6</td>
    <td>6</td>
    <td>6</td>
    <td>6</td>
    <td>6</td>
    <td>6</td>
    <td>6</td>
    <!---->
  </tr>
  <tr class="">
    <th>100</th>
    <td>6</td>
    <td>6</td>
    <td>6</td>
    <td>6</td>
    <td>6</td>
    <td>6</td>
    <td>6</td>
    <td>6</td>
    <td>6</td>
    <td>6</td>
    <td>6</td>
    <td>6</td>
    <td>6</td>
    <td>6</td>
    <td>6</td>
    <!---->
  </tr>
  <tr class="highlight-current">
    <th>105</th>
    <td>5</td>
    <td>5</td>
    <td>5</td>
    <td>5</td>
    <td>5</td>
    <td>5</td>
    <td>5</td>
    <td>5</td>
    <td>5</td>
    <td>5</td>
    <td>5</td>
    <td>5</td>
    <td>5</td>
    <td>5</td>
    <td>5</td>
    <!---->
  </tr>
  <tr class="">
    <th>110</th>
    <td>5</td>
    <td>5</td>
    <td>5</td>
    <td>5</td>
    <td>5</td>
    <td>5</td>
    <td>5</td>
    <td>5</td>
    <td>5</td>
    <td>5</td>
    <td>5</td>
    <td>5</td>
    <td>5</td>
    <td>5</td>
    <td>5</td>
    <!---->
  </tr>
  <tr class="">
    <th>115</th>
    <td>5</td>
    <td>5</td>
    <td>5</td>
    <td>5</td>
    <td>5</td>
    <td>5</td>
    <td>5</td>
    <td>5</td>
    <td>5</td>
    <td>5</td>
    <td>5</td>
    <td>5</td>
    <td>5</td>
    <td>5</td>
    <td>5</td>
    <!---->
  </tr>
  <tr class="">
    <th>120</th>
    <td>5</td>
    <td>5</td>
    <td>5</td>
    <td>5</td>
    <td>5</td>
    <td>5</td>
    <td>5</td>
    <td>5</td>
    <td>5</td>
    <td>5</td>
    <td>5</td>
    <td>5</td>
    <td>5</td>
    <td>5</td>
    <td>5</td>
    <!---->
  </tr>
  <tr class="">
    <th>125</th>
    <td>5</td>
    <td>5</td>
    <td>5</td>
    <td>5</td>
    <td>5</td>
    <td>5</td>
    <td>5</td>
    <td>5</td>
    <td>5</td>
    <td>5</td>
    <td>5</td>
    <td>5</td>
    <td>5</td>
    <td>5</td>
    <td>5</td>
    <!---->
  </tr>
  <tr class="">
    <th>130</th>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <!---->
  </tr>
  <tr class="">
    <th>135</th>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <!---->
  </tr>
  <tr class="">
    <th>140</th>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <!---->
  </tr>
  <tr class="">
    <th>145</th>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <!---->
  </tr>
  <tr class="">
    <th>150</th>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <!---->
  </tr>
  <tr class="">
    <th>155</th>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <!---->
  </tr>
  <tr class="">
    <th>160</th>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <!---->
  </tr>
  <tr class="">
    <th>165</th>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <!---->
  </tr>
  <tr class="">
    <th>170</th>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <!---->
  </tr>
  <tr class="">
    <th>175</th>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <!---->
  </tr>
  <tr class="">
    <th>180</th>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <!---->
  </tr>
  <tr class="">
    <th>185</th>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <!---->
  </tr>
  <tr class="">
    <th>190</th>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <!---->
  </tr>
  <tr class="">
    <th>195</th>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <!---->
  </tr>
  <tr class="">
    <th>200</th>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <!---->
  </tr>
  <tr class="">
    <th>205</th>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <!---->
  </tr>
  <tr class="">
    <th>210</th>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <!---->
  </tr>
  <tr class="">
    <th>215</th>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <!---->
  </tr>
  <tr class="">
    <th>220</th>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <td>4</td>
    <!---->
  </tr>
  <tr class="">
    <th>225</th>
    <td>3</td>
    <td>3</td>
    <td>3</td>
    <td>3</td>
    <td>3</td>
    <td>3</td>
    <td>3</td>
    <td>3</td>
    <td>3</td>
    <td>3</td>
    <td>3</td>
    <td>3</td>
    <td>3</td>
    <td>3</td>
    <td>3</td>
    <!---->
  </tr>
  <tr class="">
    <th>230</th>
    <td>3</td>
    <td>3</td>
    <td>3</td>
    <td>3</td>
    <td>3</td>
    <td>3</td>
    <td>3</td>
    <td>3</td>
    <td>3</td>
    <td>3</td>
    <td>3</td>
    <td>3</td>
    <td>3</td>
    <td>3</td>
    <td>3</td>
    <!---->
  </tr>
  <tr class="">
    <th>235</th>
    <td>3</td>
    <td>3</td>
    <td>3</td>
    <td>3</td>
    <td>3</td>
    <td>3</td>
    <td>3</td>
    <td>3</td>
    <td>3</td>
    <td>3</td>
    <td>3</td>
    <td>3</td>
    <td>3</td>
    <td>3</td>
    <td>3</td>
    <!---->
  </tr>
  <tr class="">
    <th>240</th>
    <td>3</td>
    <td>3</td>
    <td>3</td>
    <td>3</td>
    <td>3</td>
    <td>3</td>
    <td>3</td>
    <td>3</td>
    <td>3</td>
    <td>3</td>
    <td>3</td>
    <td>3</td>
    <td>3</td>
    <td>3</td>
    <td>3</td>
    <!---->
  </tr>
  <tr class="">
    <th>245</th>
    <td>3</td>
    <td>3</td>
    <td>3</td>
    <td>3</td>
    <td>3</td>
    <td>3</td>
    <td>3</td>
    <td>3</td>
    <td>3</td>
    <td>3</td>
    <td>3</td>
    <td>3</td>
    <td>3</td>
    <td>3</td>
    <td>3</td>
    <!---->
  </tr>
  <tr class="">
    <th>250</th>
    <td>3</td>
    <td>3</td>
    <td>3</td>
    <td>3</td>
    <td>3</td>
    <td>3</td>
    <td>3</td>
    <td>3</td>
    <td>3</td>
    <td>3</td>
    <td>3</td>
    <td>3</td>
    <td>3</td>
    <td>3</td>
    <td>3</td>
    <!---->
  </tr>
  <!---->
</tbody>
`;

exports[`Barbarian default values frenzy (first swing hits) phaseblade/colossus sword 1`] = `
<tbody>
  <tr class="highlight-current">
    <td>0</td>
    <td>8/12</td>
    <td>2.5</td>
  </tr>
  <tr class="">
    <td>4</td>
    <td>7/13</td>
    <td>2.5</td>
  </tr>
  <tr class="">
    <td>6</td>
    <td>7/12</td>
    <td>2.63</td>
  </tr>
  <tr class="">
    <td>11</td>
    <td>7/11</td>
    <td>2.77</td>
  </tr>
  <tr class="">
    <td>16</td>
    <td>7/10</td>
    <td>2.94</td>
  </tr>
  <tr class="">
    <td>23</td>
    <td>6/11</td>
    <td>2.94</td>
  </tr>
  <tr class="">
    <td>27</td>
    <td>6/10</td>
    <td>3.12</td>
  </tr>
  <tr class="">
    <td>37</td>
    <td>6/9</td>
    <td>3.33</td>
  </tr>
  <tr class="">
    <td>50</td>
    <td>6/8</td>
    <td>3.57</td>
  </tr>
  <tr class="">
    <td>60</td>
    <td>5/9</td>
    <td>3.57</td>
  </tr>
  <tr class="">
    <td>75</td>
    <td>5/8</td>
    <td>3.84</td>
  </tr>
  <tr class="">
    <td>105</td>
    <td>5/7</td>
    <td>4.16</td>
  </tr>
  <tr class="">
    <td>152</td>
    <td>5/6</td>
    <td>4.54</td>
  </tr>
  <tr class="">
    <td>174</td>
    <td>4/7</td>
    <td>4.54</td>
  </tr>
</tbody>
`;

exports[`Barbarian default values frenzy (first swing misses) 2x phaseblade 1`] = `
<tbody>
  <tr class="highlight-current">
    <td>0</td>
    <td>8.5</td>
    <td>2.94</td>
  </tr>
  <tr class="">
    <td>8</td>
    <td>8</td>
    <td>3.12</td>
  </tr>
  <tr class="">
    <td>16</td>
    <td>7.5</td>
    <td>3.33</td>
  </tr>
  <tr class="">
    <td>27</td>
    <td>7</td>
    <td>3.57</td>
  </tr>
  <tr class="">
    <td>42</td>
    <td>6.5</td>
    <td>3.84</td>
  </tr>
  <tr class="">
    <td>65</td>
    <td>6</td>
    <td>4.16</td>
  </tr>
  <tr class="">
    <td>102</td>
    <td>5.5</td>
    <td>4.54</td>
  </tr>
  <tr class="">
    <td>174</td>
    <td>5</td>
    <td>5</td>
  </tr>
</tbody>
`;

exports[`Barbarian default values standard attack 1`] = `
<tbody>
  <tr class="highlight-current">
    <td>0</td>
    <td>11</td>
    <td>2.27</td>
  </tr>
  <tr class="">
    <td>11</td>
    <td>10</td>
    <td>2.5</td>
  </tr>
  <tr class="">
    <td>26</td>
    <td>9</td>
    <td>2.77</td>
  </tr>
  <tr class="">
    <td>48</td>
    <td>8</td>
    <td>3.12</td>
  </tr>
  <tr class="">
    <td>86</td>
    <td>7</td>
    <td>3.57</td>
  </tr>
  <tr class="">
    <td>180</td>
    <td>6</td>
    <td>4.16</td>
  </tr>
</tbody>
`;
